### PR TITLE
Add a customization point to configure no-branch-deleted exit code

### PR DESCRIPTION
**Pull request by fbartho**
From my comment in https://github.com/jessfraz/branch-cleanup-action/issues/12#issuecomment-472937750 I decided to submit a patch!

Let me know if there's anything further I can do.

### CODE REVIEW

File: cleanup-pr-branch [hunk 1]
```diff
@@ -49,7 +55,7 @@ main(){
 		echo "Branch delete success!"
 		exit 0
 	fi
-	exit 78
+	exit NO_BRANCH_DELETED_EXIT_CODE
```
Comment (jessfraz) [inline cleanup-pr-branch]: does this need a $

**Comment by jessfraz** [comment]
thanks docs are always awesome!

File: cleanup-pr-branch (hunk 1 above)
Comment (fbartho) [reply cleanup-pr-branch]: You're absolutely right! I failed to copy paste correctly when I moved it from my test file into the real script.
```suggestion
	exit $NO_BRANCH_DELETED_EXIT_CODE
```

**Comment by fbartho** [comment]
Let me know if there's anything else I can do @jessfraz!

**Comment by jessfraz** [comment]
Thanks so much!

**Pull request merged**